### ISSUE TITLE
No download offered when clicking "Do some taxes"

### ISSUE THREAD

**Issue by DTB-K** [davetapley/helium-tax#79]
Hello.

After clicking "Do some taxes" I can see the numbers crunching but then no download is offered then the numbers finish crunching.

**Comment by davetapley** [comment]
@DTB-K yep it looks like this is related to Helium rate limiting their API, I will investigate further.

**Comment by davetapley** [comment]
@DTB-K okay try again now 🤞🏻

**Comment by DTB-K** [comment]
> @DTB-K okay try again now 🤞🏻

Thank you.  It seems a little intermittent still but I managed to get the export yesterday evening.  I've sent a little HNT in thanks.

**Comment by davetapley** [comment]
Glad you got it to export, thanks for the HNT.
Just as an FYI see also: https://github.com/davetapley/helium-tax/discussions/81#discussioncomment-2526514
